2019/meeting-2019-09-03.md

```diff
@@ -0,0 +1,156 @@
+        * No, same kit as last year
+
+## Motions Passed
+
+### Motion 0: Motion-passing arrangement
+
+Proposed by Rob.
+
+Background: Last week we discussed some rules for making decisions, but did not pass a decision on whether to adopt this process, as members of the committee had not had much time to read them. 
+
+Proposal: That we adopt the following rules for making decisions:
+ * All decisions are to be made via motions of the KTC.
+ * In order to be proposed, a motion must be in writing -- even if it is being made in a meeting.
+ * Only members of the KTC can propose motions.
+ * Two mechanisms for making motions:
+     * Email to kit-committee@studentrobotics.org with ``[proposal]`` in the subject line
+     * By a vote overseen by the chairperson within a meeting
+ * Meeting validity: Meetings may only be considered valid for passing motions if the following conditions are all met:
+     * At least 75% of the KTC is present (the "quorum").
+     * It is possible to attend the meeting via a teleconferencing system.
+     * 7 days notice have been provided by email to all KTC members (via kit-committee@studentrobotics.org in an email with ``[meeting]`` in the subject line)
+
+ * Decision making process in meetings:
+     * A motion may only be passed if it receives votes of acceptance from more than 50% of the KTC members that are present.
+     * In the event of a tie, the chairperson has the casting vote.
+     * If the chairperson cannot attend the meeting, then the first action within the meeting must be to appoint a temporary chairperson, who is to be approved by a vote of the quorum.
+
+ * Decision making process in email:
+     * A motion may only be passed by email if it receives votes of acceptance from _all_ of the KTC members.
+
+ * All passed motions must be recorded in the "ledger of motions".
+ * The changes that a motion describes cannot be performed or enacted upon until they are recorded in the ledger of motions.
+ * This decision making process itself can only be changed by either:
+     * A motion proposed via email, using the process described above.
+     * A motion voted on at a meeting, provided that it is accepted by more than 75% of the _KTC_ (not just the quorum for standard decisions).
+
+## Action Points
+
+* (ongoing) Rob will schedule a brainstorming session (#4)
+* (ongoing) Kier will request permission to create subteams on Github (#5)
+* (ongoing) Tom will email the people who have signed up so far (#7)
+* Rob will initiate discussion about whether we want to approve the kit team addition to the ops manual (#9)
+* Kier will tell trustees that we've reviewed the key dates doc and it looks fine
+* Tom and Rob will get container key from Rich (#10)
+* Tom and Kier will examine the inventory and estimate how many kits we can provide (#11)
+* Rob to write an email to the trustees about committee roles and brainstorming session (#12)
+
+## Appendices
+
+### Motion 1: Github permissions
```

Comment:
Perhaps add a note to say this wasn't passed?

Reply:
It wasn't rejected either. We determined that we needed to discuss it further at a later date.

Reply:
Moved it to a "Motions Not Passed" section.